py/torch_tensorrt/dynamo/conversion/impl/select.py

```diff
@@ -204,11 +246,7 @@ def index(
                 ctx, cum_adv_index, name + "_index_sum_intermediate"
             )
         else:
-            multiplier = get_trt_tensor(
-                ctx,
-                dim_tensor_list[adv_indx_indices[adv_indx_count - 1]],
-                name + "_dim_last",
-            )
+            multiplier = dim_tensor_list[adv_indx_indices[adv_indx_count - 1]]
```

Comment:
Does this change imply multiplier is always an ITensor since you were using `get_trt_tensor` before ?

Reply:
Yes we do not need `get_trt_tensor` in this, even in the previous non dynamic cases it was not required since dim_tensor_list already stores a list of dimension tensors